cmd/tailscale/cli/funnel.go

```diff
@@ -117,6 +137,29 @@ func (e *serveEnv) runFunnel(ctx context.Context, args []string) error {
 	return nil
 }
 
+func (e *serveEnv) streamFunnel(ctx context.Context, hp ipn.HostPort) error {
+	sc, err := e.lc.GetServeConfig(ctx)
+	if err != nil {
+		return err
+	}
+	if sc == nil {
+		sc = new(ipn.ServeConfig)
+	}
+	mak.Set(&sc.AllowFunnel, hp, true)
+	printFunnelWarning(sc)
```

Comment:
Since we're going to merge this in a debug state, let's remove this bit that enables Funnel as a side-effect of running stream.